usr/lib/systemd/system/greenboot-healthcheck.service

```diff
@@ -0,0 +1,17 @@
+#  This file is part of greenboot.
+#
+#  greenboot is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=greenboot Health Check Scripts Runner
+Before=greenboot.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/libexec/greenboot/greenboot.sh check
+
+[Install]
+RequiredBy=greenboot.target
```

Comment:
Hmm, this feels a bit "indirect". I would expect this dependency to happen directly in `greenboot.target` instead, e.g. something like:

```
Requires=greenboot-healthcheck.service
After=greenboot-healthcheck.service
```

?

Reply:
This would certainly work. Reasoning behind it was to keep the structure of this service just as any other, custom check service